ddtrace/profiling/collector/_threading.pyx

```diff
@@ -20,15 +20,16 @@ cpdef get_thread_name(thread_id):
     try:
         return _periodic.PERIODIC_THREADS[thread_id].name
     except KeyError:
-        # Since we own the GIL, we can safely run this and assume no change will happen,
-        # without bothering to lock anything
+        # We don't want to bother to lock anything, especially with eventlet involved, so we do a best effort to get the
```

Comment:
I think we're missing some words in the middle of this sentence:

> so we do a best effort to get the without bothering to lock anything data;

Reply:
fixed